components/other/Slider.vue

```diff
@@ -10,33 +10,31 @@
 
         <div class="carousel-inner">
           <div class="carousel-item active">
-            <img class="d-block img-fluid" src="https://s19.postimg.cc/nenabzsnn/slide4.jpg" alt="sime">
+            <img class="d-block img-fluid" src="../../assets/img/slider4.jpg" alt="sime">
             <div class="carousel-caption d-none d-md-block">
-              <h1>Mountain</h1>
+              <h3>Welcome to Joint task Foundation</h3>
+              <p>A place with your future at heart</p>
             </div>
           </div>
           <div class="carousel-item">
-            <img class="d-block img-fluid" src="https://s19.postimg.cc/nenabzsnn/slide4.jpg" alt="">
-            <div class="carousel-caption d-none d-md-block">
-              <h3>Mountain</h3>
-              <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Adipisci esse vitae exercitationem fugit,
-                numquam minus!</p>
+            <img class="d-block img-fluid" src="../../assets/img/slider7.jpg" alt="">
+            <div class="carousel-caption d-none d-md-block mb-5">
+              <h3>Vision</h3>
+              <p>It take vision to desire success. Perhaps you have a burning desire to have a better standard of living. You have a vision of a better future.</p>
             </div>
           </div>
           <div class="carousel-item">
-            <img class="d-block img-fluid" src="https://s19.postimg.cc/nenabzsnn/slide4.jpg" alt="">
+            <img class="d-block img-fluid" src="../../assets/img/slider2.jpg" alt="">
             <div class="carousel-caption d-none d-md-block">
-              <h3>Mountain</h3>
-              <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Adipisci esse vitae exercitationem fugit,
-                numquam minus!</p>
+              <h3>Unique</h3>
+              <p>Do you want to stand out from the crowd? Become the envy of many by just a few steps.</p>
             </div>
           </div>
           <div class="carousel-item">
-            <img src="https://s19.postimg.cc/nenabzsnn/slide4.jpg" alt="" class="d-block img-fluid">
+            <img src="../../assets/img/slider11.jpg" alt="" class="d-block img-fluid">
             <div class="carousel-caption d-none d-md-block">
-              <h3>Mountain</h3>
-              <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Adipisci esse vitae exercitationem fugit,
-                numquam minus!</p>
+              <h3>Team Building</h3>
+              <p>Joint task Foundation wants to partner with you to help you get to the point of your dreams. Together we can achieve it..</p>
```

Comment:
I believe this is the carousel slides, the images are still not complimentary, it mustn't be a networking image for it to pass. the overly is really not fitting in, i want it to seem natuaral

Reply:
check this sites for ideas
https://colorlib.com/preview/theme/fashi/
https://colorlib.com/preview/#instylr
https://colorlib.com/preview/#shoppers

Reply:
Great work by the way

Reply:
http://preview.themeforest.net/item/cholot-retirement-community-wordpress-theme/full_screen_preview/24092135?clickid=3H0QN5yUNxyORHlwUx0Mo3EUUki0Fv3j1VQ9y80&iradid=275988&iradtype=ONLINE_TRACKING_LINK&irgwc=1&irmptype=mediapartner&irpid=369282&mp_value1=&utm_campaign=af_impact_radius_369282&utm_medium=affiliate&utm_source=impact_radius